tests/Doctrine/Tests/Models/DDC964/DDC964Admin.php

```diff
@@ -29,7 +34,11 @@
  * })
  */
 #[Entity]
-#[AssociationOverrides([new AssociationOverride(name: 'groups', joinTable: new JoinTable(name: 'ddc964_users_admingroups'), joinColumns: [new JoinColumn(name: 'adminuser_id')], inverseJoinColumns: [new JoinColumn(name: 'admingroup_id')]), new AssociationOverride(name: 'address', joinColumns: [new JoinColumn(name: 'adminaddress_id', referencedColumnName: 'id')])])]
+#[AssociationOverrides([
+    new AssociationOverride(name: 'groups', joinTable: new JoinTable(name: 'ddc964_users_admingroups'), joinColumns: [new JoinColumn(name: 'adminuser_id')], inverseJoinColumns: [new JoinColumn(name: 'admingroup_id')]),
+    new AssociationOverride(name: 'organizations', joinTable: new JoinTable(name: 'ddc964_users_adminorganizations')),
+    new AssociationOverride(name: 'address', joinColumns: [new JoinColumn(name: 'adminaddress_id', referencedColumnName: 'id')]),
+])]
```

Comment:
Sadly, you cannot use a multiline attribute here, it will make the build fail with old versions of PHP